### ISSUE TITLE
Please incoporate new version of universalJavaApplicationStub

### ISSUE THREAD

**Issue by treimers** [fvarrui/JavaPackager#83]
I'm submitting a…

- [ ] bug report
- [x] feature request
- [ ] other

**Short description of the issue/suggestion:**

I have been running into a problem under Mac OS X with system menu bar and Java splash option. 

**Steps to reproduce the issue/enhancement:**

It seems that system menu bar and Java splash option do not work together. A detailed description of the problem can be found [here at Stackoverflow](https://stackoverflow.com/questions/65511188/javafx-problem-with-usesystemmenubar-when-starting-with-splash-option).

**What is the expected behavior?**

If no splash image is provided "universalJavaApplicationStub" should not use the splash option when invoking Java.

**What is the current behavior?**

As of version 3.0.6 the splash option is used in any case by "universalJavaApplicationStub".

**Do you have outputs, screenshots, demos or samples which demonstrate the problem or enhancement?**

See the detailed description at [Stackoverflow](https://stackoverflow.com/questions/65511188/javafx-problem-with-usesystemmenubar-when-starting-with-splash-option).

**What is the motivation / use case for changing the behavior?**

Basically this is a Java FX problem but it would be helpful to use the new version, 3.1.0, of "universalJavaApplicationStub".

**Please tell us about your environment:**

- JavaPackager version: 1.4.2
- OS version: Mac OS X 10.11.6
- JDK version: openjdk version "15" 2020-09-15, OpenJDK Runtime Environment AdoptOpenJDK (build 15+36), OpenJDK 64-Bit Server VM AdoptOpenJDK (build 15+36, mixed mode, sharing)
- Build tool: 
	- [x] Maven: Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
	- [ ] Gradle 

**Other information** (e.g. related issues, suggestions how to fix, links for us to have context)

**Comment by fvarrui** [comment]
Hi @treimers!
`universalJavaApplicationStub` script updated to v3.1.0 in JavaPackager 1.4.3-SNAPSHOT.

If it works fine for you, I'll release to Maven Central as v1.5.0, as it will include a new feature related to another issue.

**Comment by treimers** [comment]
Hi @fvarrui,

thank you very much once again. I have tested it and it works as expected.

Cheers
Thorsten

**Comment by fvarrui** [comment]
v1.5.0 released to Maven Central
